server/server_test.go

```diff
@@ -7751,6 +7756,7 @@ stream
 		.message('message')
 		.details('details')
 		.crit(lambda: TRUE)
+		.sensu()
```

Comment:
Hmm, nothing is jumping out at me either. Maybe later I'll pull down the branch and take a look.